addons/stock_landed_costs/tests/test_stock_landed_costs.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common
+from odoo.tools import convert_file
+from odoo.modules.module import get_module_resource
+
+
+@common.at_install(False)
+@common.post_install(True)
+class TestStockLandedCosts(common.TransactionCase):
+
+    def _load(self, module, *args):
+        convert_file(self.cr, 'account', get_module_resource(module, *args),
+                              {}, 'init', False, 'test', self.registry._assertion_report)
+
+    def setUp(self):
+        super(TestStockLandedCosts, self).setUp()
+
+        self._load('account', 'test', 'account_minimal_test.xml')
+
+        # In order to test the landed costs feature of stock, I create a landed cost, confirm it and check its account move created
+        self.Product = self.env['product.product']
+        self.Picking = self.env['stock.picking']
+        self.StockLandedCost = self.env['stock.landed.cost']
+        self.StockMove = self.env['stock.move']
+        self.acount_o_income = self.ref('account.o_income')
+        self.acount_o_expense = self.ref('account.o_expense')
+        self.product_uom_unit = self.ref('product.product_uom_unit')
+        self.picking_type_out = self.ref('stock.picking_type_out')
+        self.stock_location = self.ref('stock.stock_location_stock')
+        self.stock_location_customers = self.ref('stock.stock_location_customers')
+
+        # I create 2 products with different volume and gross weight and configure them for real_time valuation and real price costing method
+        self.product_1 = self.Product.create({
+            'name': 'LC Product 1',
+            'cost_method': 'real',
+            'valuation': 'real_time',
+            'weight': 10,
+            'volume': 1,
+            'property_stock_account_input': self.acount_o_expense,
+            'property_stock_account_output': self.acount_o_income})
+        self.product_2 = self.Product.create({
+            'name': 'LC Product 2',
+            'cost_method': 'real',
+            'valuation': 'real_time',
+            'weight': 20,
+            'volume': 1.5,
+            'property_stock_account_input': self.acount_o_expense,
+            'property_stock_account_output': self.acount_o_income})
```

Comment:
No need to set 
```
'property_stock_account_input': self.acount_o_expense,
'property_stock_account_output': self.acount_o_income
```
because 
```
@common.at_install(False)
@common.post_install(True)
```